docs/getting-started.md

```diff
@@ -127,18 +129,19 @@ ng build
 Now, lets build Scully and turn your Angular app into a pre-rendered static site.
 
 ```bash
-npm run scully
+npm run scully -- --nw
```

Comment:
This seems to be wrong as in this PR the option `nw` was just replaced with `watch`